### DISCUSSION TITLE
dialog中的table如何渲染数据和重新加载数据

### DISCUSSION THREAD

**Discussion by Poom6zZ** [element-plus/element-plus#16758] [Q&A]
使用可变的:key还是不能加载数据
![image](https://github.com/element-plus/element-plus/assets/56908383/35826640-3abf-4aaa-a26a-8ccad2d0729c)
![image](https://github.com/element-plus/element-plus/assets/56908383/2f1e0367-56a3-4db9-93ca-2efbf5f2beb6)
获取数据后，改变key的值来渲染
![image](https://github.com/element-plus/element-plus/assets/56908383/4d26b411-6549-450f-85e4-360509cdbead)

**Comment by warmthsea** [comment]
请在 playground 提供一个复现地址
